### DISCUSSION TITLE
Thoughts about Luau?

### DISCUSSION THREAD

**Discussion by aaronhmiller** [Kong/kong#8035] [Ideas and feature requests]
Just idle curiosity about what Kong and OpenResty devs think about [Luau](https://github.com/Roblox/luau)?

**Comment by guanlan** [comment]
@aaronhmiller Thanks for stopping by, buddy.
Our type safety choice will likely be [teal](https://github.com/teal-language/tl), it has better performance with LuaJIT support and good portability. Luau might make the most sense if embedded into the C++ projects.
